redash/destinations/asana.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import textwrap
+
+import requests
+
+from redash.destinations import BaseDestination, register
+
+
+class Asana(BaseDestination):
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "pat": {"type": "string", "title": "Asana Personal Access Token"},
+                "project_id": {"type": "string", "title": "Asana Project ID"},
+            },
+            "secret": ["pat"],
+        }
+
+    @classmethod
+    def icon(cls):
+        return "fa-sticky-note-o"
```

Comment:
The icon strings are confusing me, could you explain where this string came from and what it means?